multiqc/modules/peddy/peddy.py

```diff
@@ -5,6 +5,7 @@
 from __future__ import print_function
 from collections import OrderedDict
 import logging
+import simplejson as sj
```

Comment:
Please can you use the regular JSON import instead of adding a new dependency here. As done in other MultiQC scripts, eg:

https://github.com/ewels/MultiQC/blob/e50a2b8a9161365dea861372d03e3c3958d0ed26/multiqc/utils/util_functions.py#L7

Reply:
\* googles simplejson \*

> `json` is `simplejson`, added to the stdlib. But since `json` was added in 2.6, `simplejson` has the advantage of working on more Python versions (2.4+).
> https://stackoverflow.com/a/712799/713980

Reply:
Totally, will swap over to it.